client/src/index.css

```diff
@@ -19,3 +19,7 @@ form {
   text-align: center;
   margin: auto;
 }
+
+.text {
```

Comment:
This reminds me that we should talk about CSS architecture at some point soon.  

As written, anything with a class of `text` will get a 20px top margin.  The amount of HTML this CSS rule can impact is very large since the class is in your `index.css` file.  How do you feel about moving this rule down into a more tightly-scoped area?  Perhaps inside of a styled-component?

Reply:
I'm assuming you are talking about having a css file for every specific componnent? Yes we will chat today :)